.gitlab-ci.yml

```diff
@@ -0,0 +1,97 @@
+image: hugomrdias/node-alpine:test
+stages:
+  - check
+  - test
+  - cov
+
+variables:
+  DOCKER_DRIVER: overlay2
+
+before_script:
+  - yarn
```

Comment:
Still yarn? Just to clarify, I'd rather not use yarn in CI as I think most devs here use npm (including myself) and I want to keep the dev friction to a minimum when eventually we have a problem with npm installing a different directory structure to yarn and it causing problems.